plugins/image2/plugin.js

```diff
@@ -428,8 +428,10 @@
 
 				// Setup dynamic image resizing with mouse.
 				// Don't initialize resizer when dimensions are disallowed (http://dev.ckeditor.com/ticket/11004).
-				if ( editor.filter.checkFeature( this.features.dimension ) && editor.config.image2_disableResizer !== true )
+				// Don't initialize resizer when config.readOnly is set to true (#719).
+				if ( editor.filter.checkFeature( this.features.dimension ) && editor.config.image2_disableResizer !== true && editor.config.readOnly != true ) {
```

Comment:
What if the editor is in read-only mode because someone uses `editor.setReadOnly` method (before initializing widgets), not a configuration option?

Such possibility introduces even more problematic question: what if read-only mode would change during using an editor? I honestly don't know how to fix such issue in the current version of editor as we don't have something like "refresh" method for widgets, which can listen to any changes in the editor and react accordingly.